### ISSUE TITLE
[Tier 3:Medium] Claim/Disability Issues

### ISSUE THREAD

**Issue by Kimberley2019** [department-of-veterans-affairs/va.gov-team#6581]
**Issue Description:** Users are experiencing issues when attempting to submit and/or view claims.  Some are getting error messages, which are included in the descriptions below.

**Priority:** Medium

**Total Number of Issues:** 6

**Issue Number and User Description:**

### **IRIS: 200220-000999**

_Ever since claims status has moved to the VA.gov site I have not been able to view the status of my open claims._

https://www.va.gov/track-claims/your-claims

It does gives me the same message on my iPhone, iPad and PC.

I have tried safari, internet explorer and Firefox.
I have tried all 3 login options and have the same results. It is not due to a temporary issue. I have never been able to view my claim status on the VA.gov site.

- Initials: CH

### **IRIS: 200221-000446**

_I am unable to submit my form 21-526EZ to file an increase and secondary disability claims on va.gov. I called ebenefits and opened a work order and was sent an email yesterday afternoon advising me the problem was on va.gov end and not ebenefits. My application says "We're sorry. Something went wrong when we tried to submit your claim" when i tried to submit my claim. I have tried to re-submit my claim every day since Sunday 2/16/2020, multiple times a day with the same error. There is no error code associated with the error, just this same message that shows up._

https://www.va.gov/disability/file-disability-claim-form-21-526ez/confirmation

```
File for disability compensation
Form 21-526EZ
We’re sorry. Something went wrong when we tried to submit your claim.
For help submitting your claim:

- Please call Veterans Benefits Assistance at 800-827-1000 <+18008271000>,
Monday – Friday, 8:30 a.m. – 4:30 p.m. ET, or
- Get in touch with your nearest Veterans Service Officer (VSO). Find
out how to contact your nearest VSO
```

I am using a Dell desktop computer.

The error occurs on all browsers.

I have used DS logon, myhealthevet and id.me to log in and still have the
same problem.

I have been trying for 2 weeks to submit my claim. Please assist me in
getting this resolved.

Thank you,

- Initials: DD

### **IRIS: 200227-001066**

_When submitting a claim on va.gov I'm receiving the following error on my PC as well as mobile device:_

```
We’re sorry. We can't submit your form right now.

We’re working to fix the problem. Please make sure you’re connected to the Internet, and then try saving your form again.
Save your form.

If it still doesn’t work, please call VA Benefits and Services at 800-827-1000.
If you have hearing loss, call TTY: 711.  - Please copy and paste the the URL or web address you were on when you
experience the issue.
```

- https://www.va.gov/disability/file-disability-claim-form-21-526ez/review-and-submit
- If you saw an error message, please copy and paste the exact text
of the error message.

```
We’re sorry. We can't submit your form right now.

We’re working to fix the problem. Please make sure you’re connected
to the Internet, and then try saving your form again.

If it still doesn’t work, please call VA Benefits and Services at
800-827-1000 <18008271000>.
If you have hearing loss, call TTY: 711.
```

- What type of device are you using (i.e. dell laptop, MacBook,
iPhone, etc.)?
  - Desktop (Chrome & Internet explorer browser) & Mobile Device
(Chrome & Safari browser)
- Try a different browser (i.e. Internet Explorer, Firefox, Chrome,
etc.). Does the issue occur on all of these browsers?
  - yes
- What account do you use to sign into VA.gov (DS Logon, Myhealthevet,
ID.me)?

- Initials: MR

### **IRIS: 200216-000244**

_ I use my DS Logon to get into both VA.gov and the EBenefits websites. I can
access these sites fine. The problems arise within eBenefits when I try to
access different pages (see attached). I cannot see my disabilities,
documents, or my payment history. This has been going on since I first
accessed the site, however as you can see by the attached screenshot, it
shows that I was successful. I was told by one of your colleagues that the
issue lies with having multiple claims??? I recently received my first
disability payment, so I know it is working, but I cannot access any
information, ie. a document showing my disability status for home loan VA
funding fee exemption._

I have tried accessing from multiple desktop and laptop Windows computers,
with different browsers (Google Chrome, Internet Explorer and Firefox).
I use the link on VA.gov to access eBenefits -
https://myaccess.dmdc.osd.mil/identitymanagement/authenticate.do?execution=e1s1

Please assist. My telephone number is below as well.

- Initials: IL

### **IRIS: 200305-000003**

_I am still experiencing the issue with accessing my claims information as well as making any changes from the va.gov portal. the url is, va.gov/track-claims/your-claims. I am using a dell laptop. The problem persists when I use an iphone, Chrome or internet explorer as well as firefox. I I have also tried using a different computer and still have the same issue. I have used Ds Login and Myhealthvet to log in and still have the same issue_.

Error Message
Check your claim or appeal status
Claim status is unavailable
VA.gov is having trouble loading claims information at this time. Please check back again in a hour. Please note: You are still able to review appeals information_

-Initials: MD

### **IRIS: 200320-000902**
_I have had constant issues with seeing any of my information and have attempted to have these issues fixed to no avail. When trying to see my disability rating I have the error code: EVSS-1009. I cannot see my representative that I had. I cannot add any new disability's as the application wont load and gives me the error code: VDC-1584733961868. I cannot view payment history. I cannot see any of my additional benefits either. I hope that these can be fixed so that I can view my information. Thank you._

_URLs and Error Messages: https://eauth.va.gov/ebenefits/disabilities We couldn't retrieve your disability rating information at this time. Please try again later or call VA for assistance at 1-800-827-1000 (711 if you use a TDD), Monday - Friday 8:00am to 9:00pm ET. When requesting support, please provide the Error ID : EVSS-1009 https://eauth.va.gov/ebenefits/myDocuments We are unable to access your documents at this time. Please try again later. For access to restricted documents or documents associated with your claim that are not displayed, please contact your representative for VA claims, if you have appointed one, and he or she may be able to assist. You can also call VA at 1-800-827-1000 (711 if you use a TDD), Monday-Friday, 8:00 a.m. - 9:00 p.m. ET for assistance. When requesting support, please provide the Error ID: EVSS_6007_ _https://eauth.va.gov/ebenefits/payments https://eauth.va.gov/ebenefits/vdc?target=/wssweb/VDC526/compensation.do Application Error We're sorry, but the application has encountered an error. Please try again later or contact the VA at 1-800-827-1000. Please refer to the following error code: VDC-1585334089005 I am using an Acer laptop and I have tried both Google Chrome and Internet Explorer. I have used all login options but I mainly use my DS Login._

-Initials: JA

**Comment by luke-gcio** [comment]
Accepted the issue, will investigate.

**Comment by Mottie** [comment]
It would help if we were provided a last date and approximate time that the form was submitted.

**Comment by Kimberley2019** [comment]
This information can be requested by the Contact Center team, however, I am unsure if the users will have that exact information. Also, requesting that information could cause a delay in your research depending on how long it takes them to reply (and they still may not be able to advise us on the information that you are requesting).
Would you like for me to reach out to the users from each ticket number to see if they can provide us with that information and keep you posted non the details?

**Comment by Mottie** [comment]
We've encountered similar issues in the past, and given the generic error message that appears to the user, there isn't much we can do about troubleshooting the problem unless we get a date & time; with that information, we _might_ be able to find the error that was logged to the server.

As a backup, when a user calls in with issues, have them to submit the form while on the line. If an issue occurs, please record that date & time.

**Comment by Kimberley2019** [comment]
Hi @Mottie this is something that we can relay to the Contact Centers for the reps to try to obtain this information while on the phone with the users.  I do not have much faith that that will happen however; but I will ensure that the message gets across, that that information is valuable and needed.

**Comment by Kimberley2019** [comment]
@luke-gcio @Mottie 

The individual from ticket IRIS: 200227-001066, is still experiencing the issue.  Please see below.

Please fix this issue, i'm unable to submit the claim - this same error has 
been occurring for weeks. 

 - Please copy and paste the the URL or web address you were on when 
you experience the issue. 
 https://www.va.gov /disability/file-disability-claim-form-21-526ez/review-and-submit 

 - If you saw an error message, please copy and paste the exact text of the error message. 
We’re sorry. We can't submit your form right now. 
 We’re working to fix the problem. Please make sure you’re connected to the Internet, and then try saving your form again. If it still doesn’t work, please call VA Benefits and Services at 
 800-827-1000 18008271000.  If you have hearing loss, call TTY: 711. 
 
- What type of device are you using (i.e. dell laptop, MacBook, 
 iPhone, etc.)? 
 Desktop (Chrome & Internet explorer browser) & Mobile Device 
 (Chrome & Safari browser) 

 - Try a different browser (i.e. Internet Explorer, Firefox, Chrome, 
 etc.). Does the issue occur on all of these browsers? 
 yes 

 - What account do you use to sign into VA.gov (DS Logon, Myhealthevet, 
 ID.me)? 
 _PII Removed_

**Comment by Mottie** [comment]
I think it'd be best if we set up a call with this user.

**Comment by luke-gcio** [comment]
Agreed, can you work with Andrea to give her some personal information so she can make the first introduction with me and the team?

**Comment by Kimberley2019** [comment]
@luke-gcio I can work with @AZSchneider to get her the contact information that she needs.  Let me know how you want me to pass that along Andrea.

Also, Luke I have another issue, but will open another ticket for it (unless you want me to add it to this ticket).   It's along the same lines as the other issues in this ticket.

**Comment by Kimberley2019** [comment]
@luke-gcio @AZSchneider @Mottie @SMLuthi 
New issue added above.  Andrea, I'm going to attempt to send you a secure email with his contact information tomorrow. Unless you just want to give me a call as usual.

**Comment by Kimberley2019** [comment]
@AZSchneider I just sent you an email with the contact information for the newly added ticket in a secure document.  The second email contains the PW.

**Comment by Kimberley2019** [comment]
@luke-gcio @AZSchneider Has there been any updates as of yet?

**Comment by Kimberley2019** [comment]
@luke-gcio @AZSchneider @Mottie @SMLuthi 

Has there been any movement on this issue? If not, can the team advise on how you would like to handle this issue? If yes, can the ticket be updated with the information and closed?

**Comment by Kimberley2019** [comment]
@luke-gcio @AZSchneider @Mottie @SMLuthi

Has there been any movement on this issue? If not, can the team advise on how you would like to handle this issue? If yes, can the ticket be updated with the information and closed?

**Comment by jusweisq** [comment]
On 19 Oct 2020, from 6:35 to 6:55 pm EDT, using three operating systems with multiple browsers, the VA disability file a claim system would not accept a submission for disability compensation.  Issue described in the original post still persists. 

The website provided the following error
File for disability compensation Form 21-526EZ

"We’re sorry. Something went wrong when we tried to submit your claim."

The Source IP 173.69.155.35

Mac OS - Mozilla, Safari
Windows - Mozilla, Chrome, Edge
Android IoS - Chrome

All browser applications were updated before attempting.

**Comment by Mottie** [comment]
Hi @jusweisq!

Thanks for letting us know about this issue. It is very difficult for us to troubleshoot the problem without a specific ID that we use to save the data.

That said, there is one piece of advice I can share without knowing more: step through the application backwards, then forwards to see if the form allows progress all the way through. We have seen problems where data is changed on the last page, but it doesn't properly indicate the need for additional data in other parts of the form.

If that didn't fix the problem, please call VA Benefits and Services at 800-827-1000 to report the issue. Thanks!

**Comment by Kimberley2019** [comment]
@luke-gcio:

- Has there been any upgrades that would have fixed any of the issues that these callers have experienced?  

- Have any of these issues been revisited by your team?

- Should this ticket be closed out since/if these has not been any movement?  If so, can you please update and close out the ticket?

**Comment by Mottie** [comment]
@Kimberley2019 Our new PM is @vhenry07

**Comment by Kimberley2019** [comment]
My sincerest apologies!  Welcome @vhenry07, I cannot wait to meet you.
@Mottie thank you for the information.  Are you able to answer any of the questions posed above since your new PM may not have been looped into this as of yet?

**Comment by vhenry07** [comment]
@Kimberley2019 I'll discuss this with the team and provide update.

**Comment by Mottie** [comment]
We have made a lot of fixes to the form and have improved error handling. We've also made it easier for us to look up records that have problems. But as Ronnie said, we'll need to review these issues again and update you.

**Comment by Kimberley2019** [comment]
@vhenry07 @Mottie Thank you both very much for the prompt response, and welcome again.  I will await your update.
Make it a great Thursday!

**Comment by EvansJWang** [comment]
Hi wanted to bump this again, I am experiencing the same error

![image](https://user-images.githubusercontent.com/25827133/110688624-c5a33700-81af-11eb-93c0-e2a5b917ff9c.png)

called the number, got transfered to (no exaggeration) 4 different numbers all of which did not help.

they asked me to submit using safari, chrome and firefox, clear my cache and cookies. Still cannot submit.

**Comment by Mottie** [comment]
Sadly, that error message is very generic. We would be able to help you better if you could provide your application ID - seen in the green box (highlighted in purple in the screenshot) below the back/continue button after you make a change to any value.

![Screen Shot 2021-03-10 at 3 05 39 PM](https://user-images.githubusercontent.com/136959/110697740-60047a00-81b2-11eb-9ab9-0aa545e3a41b.png)

**Comment by Kimberley2019** [comment]
@Mottie is that something that the Contact Center group should be asking for when we receive these types of issues in order to provide it to your group?  If so, is there any other information that we should ask for that may assist your team in researching and resolving?

**Comment by Mottie** [comment]
@Kimberley2019 Yes. This number will allow us to track down the form data and any associated errors.

We're sorry about not formally providing documentation and demonstrating this application ID to your team. We can arrange for a meeting if you require one. I was only able to find this brief reference in this Slack conversation: https://dsva.slack.com/archives/CNCEXNXK4/p1606752882012100

CC: @annaswims

**Comment by EvansJWang** [comment]
@Mottie please help :)
![image](https://user-images.githubusercontent.com/25827133/110799725-77db0d00-8249-11eb-847c-d6b755460971.png)

**Comment by annaswims** [comment]
@EvansJWang I'm truly sorry for the run-around you've gotten.  I've found the mistake on our end that prevented you from submitting your application and fixed it in your form data. I'm not able to submit your form for you, but I believe that you should be able to log in and just click "submit" now. I'll also make a plan to fix this bug so that it doesn't affect more veterans in the future.

**Comment by EvansJWang** [comment]
thank you!! yeah that generic error message is tough to do anything with, I have called several times and been given many different numbers and none of them asked for the application ID number, that should definitely be updated for the help desk support!

Hope you ship that bugfix in prod soon! great work!

**Comment by Mottie** [comment]
@annaswims is the best! <img alt="ghe-emoji" title=":_happy:" src="https://cloud.githubusercontent.com/assets/136959/13559407/7996bb94-e3d9-11e5-9b36-a39832cd3ceb.gif">

**Comment by Kimberley2019** [comment]
@EvansJWang Please forgive me.  I too apologize for the run around that you had gotten about this issue.  I am also happy that you were able to get the issue resolved.  
Yes, the help desk support can be improved and it is our plan to do so.

Thank you at @annaswims and @Mottie for the information that you have provided.

@edpearson20 we need to meet regarding this and see how we can best get this information into SF for the VEO agents to utilize to obtain this information upfront.

**Comment by annaswims** [comment]
IRIS: 200320-000902 looks like eBenefits, not VA.gov

**Comment by dsickler** [comment]
I, too, am having the same issue listed here (submit disability form from VA.gov with error).  I did call the number and got bounced to 4 different people only to be told that if the problem persisted I would need to file a paper form.  Is there something I can do from my end to get this to work?  I've tried 3 different browsers across 2 different computers to no avail.

[Submission Error.docx](https://github.com/department-of-veterans-affairs/va.gov-team/files/6246902/Submission.Error.docx)

**Comment by Mottie** [comment]
That's the same error that was shared above in [this comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/6581#issuecomment-795988138). If you could please provide the ID shown in this screenshot, we can better help you

![](https://user-images.githubusercontent.com/136959/110697740-60047a00-81b2-11eb-9ab9-0aa545e3a41b.png)

**Comment by dsickler** [comment]
It is in the attachment in my comment--2902333

**Comment by dsickler** [comment]
It was in the attachment in my comment--2902333

________________________________

**Comment by annaswims** [comment]
@dsickler I believe the bug you were [affected by](https://github.com/department-of-veterans-affairs/va.gov-team/issues/22559) was fixed this afternoon. I think you should be able to submit your form now, but it you continue to have problems please let me know. I'm sorry you had to deal with this.

**Comment by dsickler** [comment]
@annaswims Thank you for looking into it!  Now I get a new error:  We’re sorry. Something went wrong when we tried to submit your claim.  No reference codes or anything.  :(  At least it is progress?

**Comment by pocketman88** [comment]
Good morning,

I wanted to post here because I am also experiencing the same issue as above.  I have attached a screenshot with the error message and my application ID # for reference.  2 calls to the contact center have yielded no success so far unfortunately.

Would anyone be able to work the same magic for my application as above by chance?  Thank you!

![image](https://user-images.githubusercontent.com/81971048/113578408-c2ee0300-95f0-11eb-844d-ad43dea7eb61.png)

**Comment by clr1121** [comment]
Hello all. This ID is having the same issue. Any way to look into it? Hours have been spent trying to get it to submit. ID: 2941577
@Mottie @annaswims

**Comment by pocketman88** [comment]
Good morning,

It looks like the above error message I posted is now gone thank you to whoever worked this.  

However, I am now unfortunately receiving a new error message.  When I click submit I receive a screen that says please wait while we process your application and provide you a confirmation number.  After this another screen comes up that states: 

![image](https://user-images.githubusercontent.com/81971048/113715507-92bd6780-96b7-11eb-8edd-213e6a9b81a3.png)

For ease of reference my application ID # is as follows: 2921921

**Comment by dsickler** [comment]
SAME!  Application ID 2902333

**Comment by vhenry07** [comment]
Acknowledging issues and understand that we have a problem related to service history. We are actively researching and working this issue and expect a resolution in 7-10 days, so closing this thread.

**Comment by dampad** [comment]
Hello, I am having issues submitting my application as well. Getting the same generic error message. But I do have my application ID: 3036880. Any help is greatly appreciated.

**Comment by againstallodds9** [comment]
> 
> 
> Acknowledging issues and understand that we have a problem related to service history. We are actively researching and working this issue and expect a resolution in 7-10 days, so closing this thread.

The issue is still occurring. Is there any new information on fixing this?

**Comment by againstallodds9** [comment]
> 
> 
> @EvansJWang I'm truly sorry for the run-around you've gotten. I've found the mistake on our end that prevented you from submitting your application and fixed it in your form data. I'm not able to submit your form for you, but I believe that you should be able to log in and just click "submit" now. I'll also make a plan to fix this bug so that it doesn't affect more veterans in the future.

@annaswims I have been running into this issue for a couple days now. My application ID is 3066567. Could you help me with this? Thank you.

**Comment by BryanDJ1995** [comment]
@annaswims hey i see you resolved the issue with the error message when trying to submit a claim for a few people, I was wondering if you could help me with this. My application ID is 3034660.

i would really appreciate it, I’ve been getting the run around like it’s no ones business, they kept redirecting me to the same numbers I had dialed it was quite frustrating, this thread is my last chance before starting a whole new application or doing it by paper.

**Comment by pocketman88** [comment]
For anyone still having an issue with this error, I managed to fix mine myself.  For me, the fix was to go into the military service history section and change the "service end date" to todays date.  I am in the reserves and am still serving so I had put my obligation end date in this section.  Once I changed it the form submitted successfully.  It appears the system is detecting errors within the package but the error message itself does not tell you what the error is for some reason.  This may not fix everyone's issue unfortunately but perhaps a few with a similar issue and could give everyone an idea of where to look for a fix.  I hope this helps in some way.

**Comment by annaswims** [comment]
Va.gov has a known bug with the system we submit claims to not accepting claims for veterans who have far-future end dates for their Guard or Reserves obligations. The system will only accept dates up to 180 days in the future.  ​

I can't officially say that anyone should enter incorrect dates, but I can say that the system will only accept dates up to 180 days in the future.

**Comment by againstallodds9** [comment]
Thank you @annaswims and @pocketman88! I was able to submit my claim.

**Comment by milewskima** [comment]
This is still an ongoing issue. I have been trying to submit my claim for application # 2828171, as recently as 7:00 EST. I retired in 2008 so I'm not having the service date issue (at least not that I see). The call center is unhelpful. Is there an ETA for resolution?

**Comment by milewskima** [comment]
Keep getting the same error as everyone above. When I called the number I was told that there was no intent to file for me. There is. I have an application number, I explained to the lady on the phone that I am unable to submit the application. I had my daughter tell her the same thing. Then I was transferred to the milConnect people because "this sounds like a DS Logon issue" and was hung up on without speaking to anyone. I read through the comments and see that this is an ongoing issue I've tried going back through the application and do not see that anything has been missed on my end. I don't have a future end date since I retired in 2008. Can someone please help me get this submitted? I have been trying since March.

![image](https://user-images.githubusercontent.com/84422377/119739064-abdccc00-be4f-11eb-99d6-862ee48ed14b.png)

**Comment by jschlieszus** [comment]
I am having the same issue as well, have tried for several weeks with no avail.  Calling the numbers is even less helpful than just repeatedly trying over and over again.

Your application has been saved. It was last saved on May 27, 2021 at 7:39 p.m. Your application ID number is 3084920.

**Comment by milewskima** [comment]
@annaswims @vhenry07 @Mottie @luke-gcio - Can anyone help or offer any suggestions on what jschileszus and I  need to do? This seems to still be an issue.

**Comment by vhenry07** [comment]
I am truly sorry you both are experiencing this issue. Our support team will be reaching out directly via email.

**Comment by jschlieszus** [comment]
Thank you! Sent from Mail for Windows 10 From: milewskimaSent: Friday, May 28, 2021 9:53 AMTo: department-of-veterans-affairs/va.gov-teamCc: jschlieszus; CommentSubject: Re: [department-of-veterans-affairs/va.gov-team] [Tier 3:Medium] Claim/Disability Issues (#6581) I am truly sorry you both are experiencing this issue. Our support team will be reaching out directly via email.I appreciate it.—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by edpearson20** [comment]
Please be informed that this particular GitHub ticket is in a "Closed" status and external comments will not be fielded appropriately.

Do note, this is an internal resource used for support, research, and development purposes.

For issues you are currently experiencing, please…

…call Veterans Benefits Assistance at 800.827.1000, Monday through Friday, 8:00 a.m. to 9:00 p.m. ET. Once you’ve reached a representative and have communicated the issue you are experiencing, please request your case number for future reference.

...or get in touch with your nearest Veterans Service Officer (VSO).

You can find an accredited representative or a VSO in 1 of 2 ways:
1) Go to eBenefits to find a local representative (including a recognized VSO, an attorney, or a claims agent) by state/territory, zip code, or the organization’s name.
 Go to eBenefits (https://www.ebenefits.va.gov/ebenefits/vso-search)
2) Or search the VA Office of the General Counsel’s list to find VA-recognized organizations and VA-accredited individuals by name, city, state, or zip code.
 Search the VA Office of the General Counsel’s list (https://www.va.gov/ogc/apps/accreditation/index.asp)

For more information, please reference “Get help filing a claim” link as shown below.
 https://www.va.gov/disability/get-help-filing-claim/

VA.Gov

**Comment by jschlieszus** [comment]
It looks like it needs to be reopened then as it isn’t resolved.  Calling VA assistance only results in them offering to show you how to manually submit.  They have no insight into the error, what causes it, and more importantly, how to fix it. Jonathan From: Ervin D Pearson IISent: Friday, May 28, 2021 3:06 PMTo: department-of-veterans-affairs/va.gov-teamCc: jschlieszus; CommentSubject: Re: [department-of-veterans-affairs/va.gov-team] [Tier 3:Medium] Claim/Disability Issues (#6581) Please be informed that this particular GitHub ticket is in a "Closed" status and external comments will not be fielded appropriately.Do note, this is an internal resource used for support, research, and development purposes.For issues you are currently experiencing, please……call Veterans Benefits Assistance at 800.827.1000, Monday through Friday, 8:00 a.m. to 9:00 p.m. ET. Once you’ve reached a representative and have communicated the issue you are experiencing, please request your case number for future reference....or get in touch with your nearest Veterans Service Officer (VSO).You can find an accredited representative or a VSO in 1 of 2 ways:Go to eBenefits to find a local representative (including a recognized VSO, an attorney, or a claims agent) by state/territory, zip code, or the organization’s name.Go to eBenefits (https://www.ebenefits.va.gov/ebenefits/vso-search)Or search the VA Office of the General Counsel’s list to find VA-recognized organizations and VA-accredited individuals by name, city, state, or zip code.Search the VA Office of the General Counsel’s list (https://www.va.gov/ogc/apps/accreditation/index.asp)For more information, please reference “Get help filing a claim” link as shown below.https://www.va.gov/disability/get-help-filing-claim/VA.Gov—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by clee3092** [comment]
Ran into this issue. Figured out that since direct deposit info was down on the site, the error was being caused by my lack of filling out the direct deposit information on the application. Best of luck to you all.
